hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -94,6 +98,8 @@ public class HandleWorkflow {
     private static final Logger logger = LogManager.getLogger(HandleWorkflow.class);
 
     public static final String ALERT_MESSAGE = "Possibly CATASTROPHIC failure";
+    // Temporary flag to control stream mode during transition to block streams
+    public static StreamMode STREAM_MODE = RECORDS;
```

Comment:
Need to delete this and use the config like how it is in develop

Reply:
Thanks, updated.